src/OpenApi/src/OpenApiGenerator.cs

```diff
@@ -460,9 +460,6 @@ private List<OpenApiParameter> GetOpenApiParameters(MethodInfo methodInfo, Endpo
         }
         else if (parameter.ParameterType == typeof(string) || ParameterBindingMethodCache.HasTryParseMethod(parameter.ParameterType))
         {
-            // complex types will display as strings since they use custom parsing via TryParse on a string
-            var displayType = !parameter.ParameterType.IsPrimitive && Nullable.GetUnderlyingType(parameter.ParameterType)?.IsPrimitive != true
```

Comment:
We need to pipe this information into Swashbuckle and NSwag now as well, likely as part of https://github.com/dotnet/aspnetcore/issues/39886.

Reply:
The `displayType` is not used anywhere in this code, that is the reason why I am removing it or maybe I am missing something.